app/src/main/java/org/projectbuendia/client/ui/BaseActivity.java

```diff
@@ -110,6 +111,22 @@ private void initializeWrapperView() {
             (FrameLayout) mWrapperView.findViewById(R.id.status_wrapper_inner_content);
         mStatusContent =
             (FrameLayout) mWrapperView.findViewById(R.id.status_wrapper_status_content);
+
+        if (snackBar == null) {
+            snackBar = new SnackBar(mWrapperView);
+        }
+    }
+
+    public int snackBar(String message) {
+        return snackBar.message(message);
+    }
+
+    public int snackBar(String message, String actionMessage, View.OnClickListener listener) {
+        return snackBar.message(message, actionMessage, listener, 999);
+    }
+
+    public int snackBar(String message, String actionMessage, View.OnClickListener listener, int priority) {
```

Comment:
It would be good to document which way priority goes - is 0 the highest or is MAX_INT the highest?